Packs/CommonScripts/Scripts/KeyValueToJson/KeyValueToJson.yml

```diff
@@ -0,0 +1,30 @@
+commonfields:
+  id: KeyValueToJson
+  version: -1
+name: KeyValueToJson
+script: ''
+type: python
+tags:
+- transformer
+- general
+comment: 'This script will take in an array of values and an array of keys. It will
+  create a JSON dictionary using the key and value pairs based on their indicies.
+  For example; value=[1, 2, 3] key=[''a'', ''b'', ''c''] will return {''a'': 1, ''b'':2,
+  ''c'': 3}.'
+enabled: true
+args:
+- name: value
+  default: true
+  description: Values to map (can be list or CSV)
+  isArray: true
```

Comment:
Please add `required:true` if the arg is mandatory. `required:false` otherwise.

Reply:
Setting this to mandatory as a transformer script often ends up with the script erroring. I always have to set the 'value' input to not required.